cmd/web-handlers.go

```diff
@@ -818,3 +789,94 @@ func presignedGet(host, bucket, object string) string {
 	// Construct the final presigned URL.
 	return host + path + "?" + query + "&" + "X-Amz-Signature=" + signature
 }
+
+// toJSONError converts regular errors into more user friendly
+// and consumable error message for the browser UI.
+func toJSONError(err error, params ...string) (jerr *json2.Error) {
+	apiErr := toWebAPIError(err)
+	switch apiErr.Code {
+	// Bucket name invalid with custom error message.
+	case "InvalidBucketName":
```

Comment:
if the caller forgets to send bucketname: `toJSONError(err)` then jerr will be `nil`